ceph_deploy/util/pkg_managers.py

```diff
@@ -235,10 +235,21 @@ def add_repo_gpg_key(self, url):
         cmd = ['apt-key', 'add', gpg_file]
         self._run(cmd)
 
+    def disable_signatures_check(self):
+        self.remote_conn.logger.info(
+            "Append APT::Get::AllowUnauthenticated 1 to /etc/apt/apt.conf"
+        )
+        self.remote_conn.remote_module.append_to_file(
+            '/etc/apt/apt.conf',
+            'APT::Get::AllowUnauthenticated 1;',
```

Comment:
this is a global setting and very unsafe to set it for an install and no set it back. Wouldn't the flag `--allow-unauthenticated` be better as a one time use case?